lib/elixir/pages/getting-started/writing-documentation.md

```diff
@@ -1,18 +1,31 @@
-# Writing documentation
-
-Elixir treats documentation as a first-class citizen. Documentation must be easy to write and easy to read. In this guide you will learn how to write documentation in Elixir, covering constructs like module attributes, style practices, and doctests.
+# Writing Documentation
 
+Elixir elevates documentation to first-class status, emphasizing ease of writing as well as readability. This guide will teach you to craft effective documentation in Elixir, covering essential constructs like module attributes, typespecs, style practices, and doctests.
```

Comment:
Let's keep the language direct. Converting "write" to "craft" makes it more indirect and imprecise for non-native speakers.